cmd/ctr/delete.go

```diff
@@ -26,13 +35,24 @@ var deleteCommand = cli.Command{
 			return errors.New("container id must be provided")
 		}
 		ctx := gocontext.TODO()
-		_, err = containers.Delete(ctx, &execution.DeleteRequest{
+		_, err = containers.Delete(ctx, &containersapi.DeleteContainerRequest{
 			ID: id,
 		})
 		if err != nil {
 			return errors.Wrap(err, "failed to delete container")
 		}
 
+		_, err = tasks.Delete(ctx, &execution.DeleteRequest{
+			ContainerID: id,
+		})
+		if err != nil {
+			// Ignore error if task has already been removed, task is
+			// removed by default after run
+			if grpc.Code(errors.Cause(err)) != codes.NotFound {
+				return errors.Wrap(err, "failed to task container")
+			}
+		}
+
 		if err := snapshotter.Remove(ctx, id); err != nil {
```

Comment:
Could you prevent this to be called on windows to avoid a panic until its snapshotter is implemented?